src/interactive/repl.ts

```diff
@@ -585,21 +590,33 @@ async function activateHere(uri: vscode.Uri) {
 
 async function activateFromDir(uri: vscode.Uri) {
     const uriPath = await getDirUriFsPath(uri)
-    await startREPL(true, false)
     if (uriPath) {
         try {
-            const activeDir = await g_connection.sendRequest<string | undefined>('repl/activateProjectFromDir', uriPath)
-            if (!activeDir) {
+            const target = await searchUpFile('Project.toml', uriPath)
+            if (!target) {
                 vscode.window.showWarningMessage(`No project file found for ${uriPath}`)
                 return
             }
-            switchEnvToPath(activeDir, true)
+            activatePath(path.dirname(target))
         } catch (err) {
             console.log(err)
         }
     }
 }
 
+async function searchUpFile(target: string, from: string): Promise<string> {
+    const parentDir = path.dirname(from)
+    // XXX: on windows ?
+    if (parentDir === from || // ensure to escape infinite recursion
+        from === '/'          // reached to the system root
```

Comment:
I'm not sure what this condition is enough for windows; any idea on this ?
```js
$ node
> const path = require('path')
> path.dirname('\\')
```
on Windows code would show what is needed here.

Reply:
I'm getting 
```
> const path = require('path')
undefined
> path.dirname('\\')
'\\'
```
on Windows, so I think we do need an additional check

Reply:
Nvm, that means the `parentDir === from` check is enough...

Reply:
Thanks for checking for me !
Now I think we actually even don't need this check:
https://github.com/julia-vscode/julia-vscode/blob/9a1a6af8a973d2e9fee3f21f1a115fb7627358b3/src/interactive/repl.ts#L621

because
```js
 node                             
Welcome to Node.js v14.6.0.
Type ".help" for more information.
> const path = require('path')
undefined
> path.dirname('/')
'/'
```

Sorry for the noise.